### PR TITLE
Remove houston reference

### PR DESCRIPTION
**Pull request by oredavids**
#### What does this PR do?
- Rewords Professional Support text.
- Removes Houston reference from FAQ, Pricing, Checkout and Contact pages.

### CODE REVIEW

**Comment by robmorgan** [comment]
@oredavids I'm not sure if anyone explicitly told you, but should we continue to mention Houston in our Terms of Service? I'm just thinking whilst we might not necessarily want to push Houston right now, we should still be covered legally.  However, feel free to ignore me if @josh-padnick or @brikis98 explicitly requested this.

**Comment by oredavids** [comment]
@robmorgan I wondered about that as well so I have a question about it in the team-website slack channel. I went ahead to include it in the PR in the event that we want to go ahead and remove it else i'll just revert that commit alone.

**Comment by robmorgan** [comment]
@oredavids okay great.

File: pages/checkout/_subscription.html [hunk 1]
```diff
@@ -103,7 +103,7 @@ <h4 class="h5 text-bold margin-top-none">Add-ons:</h4>
                   <input type="checkbox" data-switch name="pro_support">
                   <span class="slider round js-support-addon"></span>
                 </label>
-                <span class="switch-label help-text light" title="Get support via email and private chat. 2 Business Day SLA." data-toggle="tooltip">Professional Support</span>
+                <span class="switch-label help-text light" title="Get support via email or a private Slack channel. Includes 2 Business days SLA." data-toggle="tooltip">Professional Support</span>
```
Comment (brikis98) [inline pages/checkout/_subscription.html]:
```suggestion
                <span class="switch-label help-text light" title="Get support via email or a private Slack channel. Includes 2 business days SLA." data-toggle="tooltip">Professional Support</span>
```

File: pages/checkout/_subscription.html [hunk 2]
```diff
@@ -159,11 +159,10 @@ <h5 class="margin-top-none text-center no-anchor">
...
                     <li><span class="help-text light" title="Chat with fellow customers and occasionally Gruntwork engineers via Slack." data-toggle="tooltip">Community Support</span></li>
                     <li id="subscription-20-users"><span class="help-text light" title="Your subscription supports up to 20 users." data-toggle="tooltip">Up to 20 Users</span></li>
                     <li id="subscription-unlimited-users" style="display:none"><span class="help-text light" title="Your subscription supports unlimited users." data-toggle="tooltip">Unlimited Users</span></li>
-                    <li class="check-list-disabled" id="subscription-addon-1"><span class="help-text light" title="Our paid support program where you can contact us via email or private Slack. 2 Business Day SLA." data-toggle="tooltip">Professional Support</span></li>
+                    <li class="check-list-disabled" id="subscription-addon-1"><span class="help-text light" title="Our paid support program where you can contact us via email or a private Slack channel. 2 Business days SLA." data-toggle="tooltip">Professional Support</span></li>
```
Comment (brikis98) [inline pages/checkout/_subscription.html]:
```suggestion
                    <li class="check-list-disabled" id="subscription-addon-1"><span class="help-text light" title="Our paid support program where you can contact us via email or a private Slack channel. 2 business days SLA." data-toggle="tooltip">Professional Support</span></li>
```

File: _data/terms-of-service.yml [hunk 3]
```diff
@@ -33,7 +33,6 @@
         <p>If you use the Gruntwork Subscription, you also agree to the <a href="#gruntwork-subscription" title="Gruntwork Subscription Terms">Gruntwork Subscription Terms</a>.</p>
         <p>If you use the Support Tiers, you also agree to the <a href="#support" title="Support Terms">Support Terms</a>.</p>
         <p>If you use the Reference Architecture, you also agree to the <a href="#reference-architecture" title="Reference Architecture Terms">Reference Architecture Terms</a>.</p>
-        <p>If you use Gruntwork Houston, you also agree to the <a href="#gruntwork-houston" title="Gruntwork Houston Terms">Gruntwork Houston Terms</a>.</p>
```
[the remark this answers is not in the document]
Comment (josh-padnick) [reply _data/terms-of-service.yml]: I requested Ore do the opposite, but you do make a fair point. Ok, let's leave them as is for now.

[the remark this answers is not in the document]
Comment (oredavids) [reply _data/terms-of-service.yml]: OK

**Pull request merged**